Python/bltinmodule.c

```diff
@@ -1429,10 +1447,13 @@ map_next(mapobject *lz)
     }
 
     Py_ssize_t nargs = 0;
-    for (Py_ssize_t i=0; i < niters; i++) {
+    for (i=0; i < niters; i++) {
```

Comment:
It's preferred to align modified code with PEP-7.

```suggestion
    for (i = 0; i < niters; i++) {
```

Reply:
We use `i=0` in many places including this module.  PEP 7 is not specific this and recommends mostly matching other code in the mode.  In this case, the code is being modeled after `zip()`.